### ISSUE TITLE
The updated package list in GitHub Release is a binary

### ISSUE THREAD

**Issue by trivikr** [aws/aws-sdk-js-v3#3484]
## Describe the bug
The updated package list in GitHub Release is a binary

#### SDK version number
Release automation

#### Is the issue in the browser/Node.js/ReactNative?
N/A

#### Details of the browser/Node.js/ReactNative version
N/A

## Steps to reproduce
Download [Updated_packages.md](https://github.com/aws/aws-sdk-js-v3/releases/download/v3.56.0/Updated_packages.md) from v3.56.0 release, and open it.

#### Observed behavior
VSCode Version: 1.65.2 shows the following message:
```
The file is not displayed in the editor because it is either binary or uses an unsupported text encoding.
```

<details>
<summary>Screenshot</summary>

![Screen Shot 2022-03-25 at 10 08 04 AM](https://user-images.githubusercontent.com/16024985/160168374-b3be79f4-a7fa-428f-b559-3bc4f9ea9f72.png)

</details>

On clicking "Do you want to open it anyway?", it shows binary data.

<details>
<summary>Screenshot</summary>

![Screen Shot 2022-03-25 at 10 09 29 AM](https://user-images.githubusercontent.com/16024985/160168530-be94703b-74ae-4abb-b782-e8e371fdb0dc.png)

</details>

#### Expected behavior
The file opens in text format in the editor and provides Markdown preview option to the user if supported by editor.

**Comment by trivikr** [comment]
Internal ticket P61853276

**Comment by trivikr** [comment]
Update: The file isn't a binary, but a zip file. Verified by:
* Downloading https://github.com/aws/aws-sdk-js-v3/releases/download/v3.56.0/Updated_packages.md
* Rename `Updated_packages.md` to `Updated_packages.zip`
* Extract `Updated_packages.zip` which gives `updated-packages.md` which has list of updated packages.

**Comment by trivikr** [comment]
We decide to rename folder to `assets-{version}.zip` instead, fixed in internal CR-67315110.
This issue can be resolved after verifying `assets-3.64.0.zip` in v3.64.0 release.

**Comment by trivikr** [comment]
Verified that file is renamed to `assets-{version}.zip` in https://github.com/aws/aws-sdk-js-v3/releases/tag/v3.64.0

The zip can be downloaded on local machine, and the file **updated-packages.md** has list of updated packages.
